tests/Fixer/ListNotation/ListSyntaxFixerTest.php

```diff
@@ -88,6 +88,17 @@ public function provideToLongCases()
 ;#',
         ];
 
+        $cases[] = ['<?php
+
+class Test
+{
+    public function updateAttributeKey($key, $value)
+    {
+        $this->{camel_case($attributes)}[$key] = $value;
```

Comment:
Is `camel_case` intentional? Got me confused seeing words "camel case" in snake case ;)

Reply:
it is a copy paste of the reported sample that failed, I hadn't notice this until you pointed it out ;)

Reply:
Sorry, was a copy + paste from our Laravel app. `camel_case()` is a helper function. ^^;